src/language/sv.json

```diff
@@ -75,7 +75,7 @@
     "row-error": "Virhe rivillä SV?",
     "file-read-error": "Tiedoston lukemisessa tapahtui odottamaton virhe. SV?"
   },
-  "units": { "year": "år", "pieces": "St", "celsius": "°C" },
+  "units": { "days": "dagar", "year": "år", "pieces": "St", "celsius": "°C" },
```

Comment:
Näitä ei enää sinällään tarvi sv.jsoniin lisätä kun meillä on scripti ylläpitämään tuota osaa.

Reply:
Kääntäisikö skripti sen myös? Kyllähän nämä varmaan jossain vaiheessa menevät ihan ammattikääntäjälle, mutta ajattelin että näitä yksinkertaisia tapauksia olisi ihan hyvä laittaa samantien, niin ovat aiemmin paikallaan.

Reply:
Tästä dailyssa tarkennettiin, että niitä SV? -loppuisia placeholdereita ei tarvitse lisäillä, mutta on hyödyllinen lisätä jos tekee samalla käännöksen valmiiksi.